src/_includes/components/testimonials-slider.html

```diff
@@ -2,7 +2,7 @@
     <div class="c-slider__slides-container" data-slides>
         <div class="c-slider__slides-wrapper" data-slides-wrapper>
             {%- for item in site.donate_page.testimonials.items -%}
-            <div class="c-slider__slide focusable" data-slide>
+            <div class="c-slider__slide" data-slide>
```

Comment:
<img width="1422" alt="Screenshot 2022-09-14 at 9 49 41 PM" src="https://user-images.githubusercontent.com/32865581/190208566-9df158e2-a96b-4c57-8383-0aad9819ef73.png">
I still see the focus in your deploy preview can you check this?  or is this expected?

Reply:
I left it out. I'm not sure which node is affecting this behavior right now, and I'm going to wake up tomorrow and check to see.